### DISCUSSION TITLE
Differences in nightly release for Windows ?

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#3931] [General]
Dear @rustdesk 
looking at the nightly release I see these two: rustdesk-1.2.0-x86_64-pc-windows-msvc.exe  and   rustdesk-1.2.0-x86_64.exe  what is the difference between them?
What is the most properly to install for Windows 10 , Windows 11 PCs ?

Many thanks

**Comment by WhiteAls** [comment]
> Dear @rustdesk looking at the nightly release I see these two: rustdesk-1.2.0-x86_64-pc-windows-msvc.exe and rustdesk-1.2.0-x86_64.exe what is the difference between them? What is the most properly to install for Windows 10 , Windows 11 PCs ?
> 
> Many thanks

here is why #3920
the longer one (rustdesk-1.2.0-x86_64-pc-windows-msvc.exe) is just old nightly that needs to be deleted

**Reply by mikekgr** [reply]
Thanks a lot. So for now on rustdesk-1.2.0-x86_64.exe is the correct way to go?
